src/rbus/rbus_interval_sub.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
rename the file as,
rbus_intervalsubscription.c
rbus_intervalsubscription.h

Reply:
file names are changed as per suggestion